packages/ui-extensions/src/surfaces/customer-account/components/Avatar/Avatar.doc.ts

```diff
@@ -16,7 +16,7 @@ const data: ReferenceEntityTemplateSchema = {
       filePath: 'components/Avatar/Avatar.ts',
     },
   ],
-  category: 'Components',
+  category: 'components',
```

Comment:
why do we have to do this?

Reply:
It's because the feature to customize categories was case sensitive, but they have pushed a fix since.
I still decided to keep it as lowercase, since it does not affect the rendering.